TTHAnalysis/python/plotter/chargeFlips/RooCMSShape.h

```diff
@@ -0,0 +1,94 @@
+/*****************************************************************************
```

Comment:
@stiegerb TTHAnalysis/python/plotter/chargeFlips/RooCMSShape.h should not be here, but taken from the release using the $CMSSW_RELEASE_BASE environment variable